rasterio/rio/info.py

```diff
@@ -99,6 +122,48 @@ def info(ctx, input, aspect, indent, namespace, meta_member, verbose, bidx,
                         float(band.mean())))
                 elif meta_member == 'checksum':
                     click.echo(str(src.checksum(bidx)))
+
+                elif meta_member == 'gcps':
+                    gcps, crs = src.gcps
+
+                    if geojson_type == 'collection':
+                        features = [p.__geo_interface__ for p in gcps]
```

Comment:
Since the coordinates get transformed to 4326 by default, what about leaving the projected x and y in the feature properties in case user's need to refer back to the original coordinates?